main_server/local_server/server_admin/c_comdef_admin_main_console.class.php

```diff
@@ -1818,6 +1818,10 @@ function return_single_meeting_format_template()
             $ret .= '<div class="format_tab_inner_div">';
                 $f_array = $this->my_server->GetFormatsArray();
                 $f_array = $f_array[$this->my_server->GetLocalLang()];
+                usort($f_array, function ($a, $b) {
+                    if ($a->GetKey() == $b->GetKey()) return 0;
```

Comment:
Dude, nice. 

Per PSR-2, will you put braces around the body of your if statement? Wanna try to get some good coding practices going in this repo.

Reply:
cool, should i just use string comapre? 
return strcmp($a->GetKey(), $b->GetKey());

Reply:
Yeah probably